lib/elixir/lib/stream/reducers.ex

```diff
@@ -120,6 +120,18 @@ defmodule Stream.Reducers do
     end
   end
 
+  defmacro map_every(nth, mapper, f \\ nil) do
+    quote do
+      fn
+        (entry, acc(h, n, t)) when n === :first
+                              when n === unquote(nth) ->
+          next_with_acc(unquote(f), unquote(mapper).(entry), h, 1, t)
+        (entry, acc(h, n, t)) ->
+          skip(acc(h, n+1, t))
```

Comment:
I believe this would be the correct expression:

    next_with_acc(unquote(f), entry, h, n+1, t)